libsplinter/src/biome/profile/store/mod.rs

```diff
@@ -70,7 +71,7 @@ impl Profile {
     }
 }
 
-/// Builder for profile.
+/// Builder for profile, user_id and subject are required
```

Comment:
I would suggest putting this on a separate line - it's a detail and the first line is the summary in the list of structs in the doc.

I would also add a note in the `with_` methods for the required items on the order of `This is a required field for the final Profile struct.`

Reply:
Fixed